.github/release_plan.md

```diff
@@ -33,7 +33,6 @@
         -   [ ] `package.json`
         -   [ ] `package-lock.json`
         -   [ ] Confirm the 3rd part of the version ends with `100`.
-    -   [ ] Update the VS Code engine version in package.json (for release version) to target the upcoming stable release. This will allow us to release before VS Code and be ready in the marketplace. [Example](https://github.com/microsoft/vscode-jupyter/pull/9494/files)
```

Comment:
this still happens at some point right?

Reply:
This is done when the next insiders has been shipped (here's the section where its done)
```
-   [ ] Bump the engines.vscode version on the `main` branch to point to the next insider version. For example, from `1.58.0-insider` to `1.59.0-insider`
```